api/controllers/FlairController.js

```diff
@@ -178,11 +178,11 @@ module.exports = {
         var ipAddress = req.headers['x-forwarded-for'] || req.ip;
         Event.create([{
           type: "flairTextChange",
-          user: req.user.name,
+          userName: req.user.name,
```

Comment:
Ah right, yeah I guess it would. It just feels annoying to me. I had assumed your script migrated the events.